packages/control-property-editor/src/App.tsx

```diff
@@ -87,29 +89,45 @@ export default function App(appProps: AppProps): ReactElement {
         setWarningDialogVisibility(false);
     }
 
+    useEffect(() => {
+        if (dialogMessage && isAdpProject) {
+            setShouldShowDialogMessageForAdpProjects(true);
+        }
+    }, [dialogMessage]);
```

Comment:
This is not working as expected. It is executed before actually the scenario is set to `ADAPTATION_PROJECT`. You can see it best when debugging. Please have a look.